src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -162,6 +163,23 @@ public Response delete(@PathParam("email") String email, @Context UriInfo info)
         return Response.ok().build();
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/SigningOfficials")
+    @RolesAllowed(RESEARCHER)
+    public Response getSOsForInstitution(@Auth AuthUser authUser) {
+        try {
+            User user = userService.findUserByEmail(authUser.getName());
+            if (Objects.isNull(user.getInstitutionId())) {
+                throw new NotFoundException("Current user, " + user.getDisplayName() + ", does not have an institution.");
+            }
+            List<User> signingOfficials = userService.findSOsByInstitutionId(user.getInstitutionId());
+            return Response.ok().entity(signingOfficials).build();
```

Comment:
I'm nervous about the security impact here. We're opening up a lot of information about the signing official to all researchers. Let's see if we can trim this down to a list of display names and ids instead of full objects.

Reply:
That's a good idea, display names and user ids? or we could do display names and emails, or just display names, if we don't want to possibly expose the userId. I can just update the underlying query to only select for those fields and the rest of the fields will be null. Does that sound good? I can also create a new object in the service class if we don't want to pass along the null values